HTML/mockup-ellipsis/index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
+    <title>Mockup 2</title>
+</head>
+
+<body>
+    <header class="header">
```

Comment:
ratio of image should not depend on window size
![image](https://user-images.githubusercontent.com/11594423/59268624-68cac380-8c4d-11e9-904d-32b5d49982c8.png)


![image](https://user-images.githubusercontent.com/11594423/59268751-afb8b900-8c4d-11e9-83d0-fab4973d6149.png)